tools/ips.py

```diff
@@ -4,30 +4,32 @@
 # writes that list to the Rust file.
 # When testing is done, just change --op to 'remove' and run the script again to remove all the dummy devices (or aliases).
 #
-# Linux and Mac supported! Script need to be run with sudo.
+# Linux and Mac supported! User must be in sudoers file to use this script.
```

Comment:
Should we move this script to the ziggurat-core repo?

Reply:
Well, we could but it would demand user to fetch core repo, extract ips.py file, copy it to folder with repository and then run it. Till we think out the easier way I'd leave it here.